go.mod

```diff
@@ -2,4 +2,8 @@ module github.com/cloudflare/tableflip
 
 go 1.14
 
-require golang.org/x/sys v0.0.0-20210119212857-b64e53b001e4
+require (
+	golang.org/x/sys v0.0.0-20210119212857-b64e53b001e4
+	google.golang.org/grpc v1.49.0
```

Comment:
That's a heavy dependency just for 1 example, better to keep in a separate module.

Reply:
> That's a heavy dependency just for 1 example, better to keep in a separate module.

Thank you for your feedback and suggestions. To address this, I will create a separate module for the example code to keep the dependencies isolated. 

Additionally, I think that **http_example_test.go** and **tcp_example_test.go** could also be included in this module. I'm not sure if this would be appropriate though.

Have a nice day!